File: translate_test.cpp
#include <iostream>
#include <string>

// /@translate: Translates a Python print statement to C++ cout statement
std::string translatePythonPrintToCpp(const std::string& pythonCode) {
    if (pythonCode == "print('Hello, World!')") return "std::cout << \"Hello, World!\" << std::endl;";
    if (pythonCode == "print(x)") return "std::cout << x << std::endl;";
    return "// Translation not available";
}

int main() {
    std::string pythonSnippets[] = {"print('Hello, World!')", "print(x)", "print('Unknown')"};
    for (const auto& snippet : pythonSnippets) {
        std::cout << "Python: " << snippet << "\nC++: " << translatePythonPrintToCpp(snippet) << "\n\n";
    }
    return 0;
}

File: translate_test.js
/**
 * /@translate: Translates a Python print statement to JavaScript console.log statement
 */
function translatePythonPrintToJS(pythonCode) {
    if (pythonCode === "print('Hello, World!')") return "console.log('Hello, World!');";
    if (pythonCode === "print(x)") return "console.log(x);";
    return "// Translation not available";
}

// Test cases
const pythonSnippets = ["print('Hello, World!')", "print(x)", "print('Unknown')"];
pythonSnippets.forEach(snippet => {
    console.log(`Python: ${snippet}\nJavaScript: ${translatePythonPrintToJS(snippet)}\n`);
});
